cmd/cloud-controller-manager/main.go

```diff
@@ -82,6 +81,16 @@ func main() {
 		Constructor: nodeIpamController.startNodeIpamControllerWrapper,
 	}
 
+	gkeServiceController := gkeServiceController{}
+	gkeServiceController.gkeServiceControllerOptions.AddFlags(cloudProviderFS)
```

Comment:
nit: Can this be transformed into a single `NewGkeServiceController(cloudProviderFS)`